project-ideas/shopify-integration/orders/createShopifyOrder.md

```diff
@@ -0,0 +1,156 @@
+# Developer Notes – process#ShopifyOrders Service
+
+Below are the functional notes describing how each part of the service processes Shopify REST + GraphQL order fields and stores them into the OMS (OFBiz).
+
+## 1. Order Existence Check
+
+**Purpose:** Prevent duplicate order creation.  
+**Logic:**
+
+- Check `OrderIdentification` for `SHOPIFY_ORD_ID = legacyResourceId`.
```

Comment:
I don't personally like that we have to check a vertical table rather than just checking external id. why not just Shopify shop order table?